github/release_notes.py

```diff
@@ -14,26 +14,81 @@
 
 import re
 from collections import namedtuple
-
 import git
 from git.exc import GitError
 from pydash import _
 from semver import parse_version_info
 from github.util import GitHubRepositoryHelper
 
 from util import info, warning, fail, verbose, existing_dir
+from product.model import ComponentReference
+from model.base import ModelValidationError
+
+ReleaseNote = namedtuple('ReleaseNote', [
+    "category_id",
+    "target_group_id",
+    "text",
+    "reference_is_pr",
+    "reference_id",
+    "user_login",
+    "origin_repo",
+    "is_current_repo",
+    "component_ref"
+])
+
+def create_release_note_obj(
```

Comment:
seeing that you need some non-trivial c'tor, would it not make more sense to define a regular python class ReleaseNote and implement a c'tor? The amount of required code should not differ by much

Reply:
I also had a class (with __slots__) for it but then also had to write the __eq__ and __hash__ methods for it which also would require a decent amount of code lines (except I missed something) and I figured it's easier to go this way